### ISSUE TITLE
Annotations UUID

### ISSUE THREAD

**Issue by CorreyL** [CorreyL/qupath-annotation-exchange#3]
Annotations made and exported from `QuPath` currently do not get assigned a `UUID` (unlike when annotations are made in the internal GSC app):

Specifically, when annotations originally made in `QuPath` are re-imported, they are just assigned incremented numbers:

![image](https://user-images.githubusercontent.com/16601729/70575792-12870080-1b5c-11ea-99d5-800002abc7aa.png)

Give the name/label for each annotation a consistent naming convention between this QuPath extension, and the internal GSC app.

**Comment by CorreyL** [comment]
Partially fixed in [PR#6](https://github.com/CorreyL/qupath-annotation-exchange/pull/6), wherein the name of each annotation is retrieved and assigned to the `uid` and `name` attributes.

This fixes the issue when annotations are imported, however a unique `uid` and `name` need to be generated if the annotations originate from `QuPath`.

**Comment by CorreyL** [comment]
Fixed in PR#6, specifically commit e5569a4

I made some annotations in `QuPath`:

![image](https://user-images.githubusercontent.com/16601729/73675866-1caf1200-4668-11ea-9a89-430478ca7354.png)

And exported the annotations using the latest build (e5569a4):

```json
{
  "SourceSlide": "24496.svs",
  "dictionaries": [{
    "uid": "ff944894-a10b-41da-8ca5-a18dff9cd8e6",
    "name": "ff944894-a10b-41da-8ca5-a18dff9cd8e6",
    "path": {
      "applyMatrix": true,
      "segments": [
        [
          [5029.78173828125, 5452.5673828125],
          [0.0, 0.0],
          [0.0, 0.0]
        ],
        [
          [3436.33740234375, 7967.2216796875],
          [0.0, 0.0],
          [0.0, 0.0]
        ],
        [
          [6797.5087890625, 8365.5830078125],
          [0.0, 0.0],
          [0.0, 0.0]
        ],
        [
          [7419.9482421875, 6473.36767578125],
          [0.0, 0.0],
          [0.0, 0.0]
        ],
        [
          [5029.78173828125, 5452.5673828125],
          [0.0, 0.0],
          [0.0, 0.0]
        ]
      ],
      "closed": true,
      "strokeColor": [1.0, 0.0, 0.0],
      "fillColor": [1.0, 0.0, 0.0, 0.5],
      "strokeScaling": false
    },
    "zoom": 1.0,
    "context": [],
    "dictionary": "imported",
    "label": "Tumor"
  }, {
    "uid": "8ccc2679-283f-4fdd-a6a6-6b67bff18a8a",
    "name": "8ccc2679-283f-4fdd-a6a6-6b67bff18a8a",
    "path": {
      "applyMatrix": true,
      "segments": [
        [
          [13594.544921875, 5128.89892578125],
          [0.0, 0.0],
          [0.0, 0.0]
        ],
        [
          [11229.2763671875, 7768.041015625],
          [0.0, 0.0],
          [0.0, 0.0]
        ],
        [
          [15412.0673828125, 8664.353515625],
          [0.0, 0.0],
          [0.0, 0.0]
        ],
        [
          [13594.544921875, 5128.89892578125],
          [0.0, 0.0],
          [0.0, 0.0]
        ]
      ],
      "closed": true,
      "strokeColor": [1.0, 0.0, 0.0],
      "fillColor": [1.0, 0.0, 0.0, 0.5],
      "strokeScaling": false
    },
    "zoom": 1.0,
    "context": [],
    "dictionary": "imported",
    "label": "Tumor"
  }]
}
```
